pandas/io/excel/_openpyxl.py

```diff
@@ -453,6 +459,8 @@ def write_cells(
             self.sheets[sheet_name] = wks
 
         if validate_freeze_panes(freeze_panes):
+            # for mypy
+            assert freeze_panes is not None
```

Comment:
not sure if its been discussed, but i think we should move away from the assert-for-mypy pattern, since with cast/ignore mypy will alert us when it is no longer necessary

Reply:
Will use cast then